### PR TITLE
datePicker color update

### PR DESCRIPTION
**Pull request by yananym**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

DatePicker: Updated selection styles for date, selected date range and font colors in Calendar

#### Focus areas to test

DatePicker

### CODE REVIEW

**Comment by erichdev** [comment]
@dzearing @christiango @micahgodbolt I'm curious if any of you accepted the change on Screener. If not, I wonder why the regression wasn't flagged here.

File: packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss [hunk 1]
```diff
@@ -28,7 +28,6 @@ $Calendar-dayPicker-margin: 10px;
   display: none;
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss]: missing change file

Comment (yananym) [reply packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss]: committed, not pushed 😬

**Comment by johannao76** [comment]
Can you send before/after screenshot of the changes? Thanks!

**Comment by erichdev** [comment]
@yananym Could you update the snapshots to get the build passing? Or I think if you rebase master that should get the tests to pass (snapshots were updated in another PR that was merged). 

Once the build passes, Screener will provide screenshots of changes. But if there are any relevant screenshots that don't post to Screener for some reason, can you provide them for @johannao76 so we can get this PR moving along? Thanks!

**Comment by micahgodbolt** [comment]
Travis is failing due to a change in the DOM for this component

image

Though from what I see in this PR, i'm not sure why it's happening. Can you cd into packages/office-ui-fabric-react and run `npm run update-snapshots` and see if you find the same changes?

**Pull request closed without merging**